--- gnome-main-menu-1.8.0/main-menu/src/main-menu-ui.c~	2014-02-11 19:01:39.000000000 +0000
+++ gnome-main-menu-1.8.0/main-menu/src/main-menu-ui.c	2014-10-21 17:07:57.726493754 +0000
@@ -1486,7 +1486,7 @@
 	argv = g_new0 (gchar *, argc + 1);
 
 	for (i = 0; i < argc; ++i) {
-		argv [i] = g_strdup_printf (argv_parsed [i], (search_txt == NULL) ? "" : search_txt);
+		argv [i] = g_strdup_printf ( "%s", argv_parsed [i], (search_txt == NULL) ? "" : search_txt);
 	}
 
 	argv [argc] = NULL;
